README.md

```diff
@@ -15,47 +15,110 @@
 
 Manticore is a symbolic execution tool for analysis of smart contracts and binaries.
 
-> Note: Beginning with version 0.2.0, Python 3.6+ is required.
-
 ## Features
 
-- **Input Generation**: Manticore automatically generates inputs that trigger unique code paths
-- **Error Discovery**: Manticore discovers bugs and produces inputs required to trigger them
-- **Execution Tracing**: Manticore records an instruction-level trace of execution for each generated input
+- **Program Exploration**: Manticore can execute a program with symbolic inputs and explore all the possible states it can reach
+- **Input Generation**: Manticore can automatically produce concrete inputs that result in a given program state
+- **Error Discovery**: Manticore can detect crashes and other failure cases in binaries and smart contracts
+- **Instrumentation**: Manticore provides fine-grained control of state exploration via event callbacks and instruction hooks
 - **Programmatic Interface**: Manticore exposes programmatic access to its analysis engine via a Python API
 
 Manticore can analyze the following types of programs:
 
 - Ethereum smart contracts (EVM bytecode)
-- Linux ELF binaries (x86, x86_64 and ARMv7)
+- Linux ELF binaries (x86, x86_64, aarch64, and ARMv7)
+- WASM Modules
+
+## Installation
+
+> Note: We recommend installing Manticore in a [virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#installing-virtualenv)
+ to prevent conflicts with other projects or packages
+
+Option 1: Installing from PyPI:
+
+```bash
+pip install manticore
+```
+
+Option 2: Installing from PyPI (with extra native dependencies):
```

Comment:
Why would someone want `native`? Might be nice to put an answer to that here.

Reply:
Rephrased to make this more clear